charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -273,6 +273,11 @@ data:
           - key: type
             action: delete
 
+      resourcedetection/cloudproviders:
+        detectors: [env, gcp, eks, azure, aks, ec2, ecs]
```

Comment:
You'll need to add `aks` to the `ignore_words_list` in https://github.com/newrelic/helm-charts/blob/6a0e3d0081da77dcd7da0fdffe57eceb37c81f45/.github/workflows/lint_test_charts.yaml#L17

Reply:
good catch, thanks